src/importexport/mei/internal/meiexporter.cpp

```diff
@@ -817,6 +817,8 @@ bool MeiExporter::writeMeasure(const Measure* measure, int& measureN, bool& isFi
             success = success && this->writeHairpin(dynamic_cast<const Hairpin*>(controlEvent.first), controlEvent.second);
         } else if (controlEvent.first->isHarmony()) {
             success = success && this->writeHarm(dynamic_cast<const Harmony*>(controlEvent.first), controlEvent.second);
+        } else if (controlEvent.first->isArticulation() && !controlEvent.first->isOrnament()) {
+            success = success && this->writeLv(dynamic_cast<const Articulation*>(controlEvent.first), controlEvent.second);
```

Comment:
Still one question. When an articulation is not an ornament, that doesn't necessarily mean that it is LV tie. It might also be a staccato or tent or …. Is that handled correctly by this code?

Reply:
Ah wait, now I see that probably works because only ornaments and LV ties make it into the control event map (MeiExporter::fillControlEventMap). Would be good to make this a bit clearer, because now someone might update one place but forget to update the other.

Reply:
@cbjeukendrup Yeah, it looks a bit messy. I wanted to keep the alphabetical order of the write* calls. I've added some comments.